src/libs/kata-types/src/config/drop_in.rs

```diff
@@ -0,0 +1,214 @@
+// Copyright (c) 2022 Red Hat
```

Comment:
Not really an upstream consideration, but fwiw, I believe Red Hat Legal recommends using simply `Copyright Red Hat` rather than putting years or `(c)` in such notices (I was surprised too).